pom.xml

```diff
@@ -318,11 +316,11 @@
 					<serverId>alfresco-internal-staging</serverId>
 					<nexusUrl>https://artifacts.alfresco.com/nexus</nexusUrl>
 					<stagingProfileId>${release.staging.profile}</stagingProfileId>
-					<stagingDescription>Staging Share ${project.version} (${bamboo_buildResultKey} ${bamboo_planRepository_revision})</stagingDescription>
+					<stagingDescription>Staging Share ${project.version} (${build-key}-${build-number} ${scm-revision})</stagingDescription>
 					<tags>
-						<build.number>${bamboo_buildNumber}</build.number>
-						<build.key>${bamboo_buildKey}</build.key>
-						<build.revision>${bamboo_planRepository_revision}</build.revision>
+                        <build.number>${build-number}</build.number>
```

Comment:
Some will complain about the formatting here

Reply:
Thank you for spotting that, the piece of configuration was copied from 5.2.7 Branch and my code formatter did some magic with it, I`ve corrected this.

Reply:
👍🏼